### PR TITLE
Update deploy-tidb-lightning.md

### PR DESCRIPTION
**Pull request by Jingyuma**
[WARN] [base_client.go:181] ["[pd] failed to get cluster id"] [url=http://basic-pd:2379] [error="[PD:client:ErrGetClusterID] fail to get cluster id"] [cause="error:rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp: lookup basic-pd: Temporary failure in name resolution\" target:basic-pd:2379 status:TRANSIENT_FAILURE"]

### What is changed, added or deleted? (Required)

add PD address parameter when use local backend

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@Jingyuma Thanks for your contribution! Can you sign the [CLA](https://github.com/pingcap/docs-cn/pull/4556#issuecomment-696022376) for this PR to be merged?
@glorv PTAL and confirm all applicable versions, thanks!

**Comment by glorv** [comment]
LGTM

thanks @Jingyuma

**Comment by glorv** [comment]
> @glorv PTAL and confirm all applicable versions, thanks!
This should be apply to `master` and `release-4.0` @TomShawn

File: tidb-lightning/deploy-tidb-lightning.md [hunk 1]
```diff
@@ -124,8 +124,12 @@ TiDB Lightning 可随 TiDB 集群一起用 [TiDB Ansible 部署](/online-deploym
     backend = "local"
     # 设置排序的键值对的临时存放地址，目标路径需要是一个空目录
     "sorted-kv-dir" = "/mnt/ssd/sorted-kv-dir"
+    
+    [tidb]
+    # pd-server 的地址，填一个即可。
```
Comment (TomShawn) [inline tidb-lightning/deploy-tidb-lightning.md:129]:
```suggestion
    # pd-server 的地址，填一个即可
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 2]
```diff
@@ -181,6 +185,8 @@ TiDB Lightning 可随 TiDB 集群一起用 [TiDB Ansible 部署](/online-deploym
     password = ""
     # 表架构信息在从 TiDB 的“状态端口”获取。
     status-port = 10080
+    # pd-server 的地址，填一个即可。
```
Comment (TomShawn) [inline tidb-lightning/deploy-tidb-lightning.md:188]:
```suggestion
    # pd-server 的地址，填一个即可
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 3]
```diff
@@ -124,8 +124,12 @@ TiDB Lightning 可随 TiDB 集群一起用 [TiDB Ansible 部署](/online-deploym
     backend = "local"
     # 设置排序的键值对的临时存放地址，目标路径需要是一个空目录
     "sorted-kv-dir" = "/mnt/ssd/sorted-kv-dir"
+    
+    [tidb]
+    # pd-server 的地址，填一个即可
+    pd-addr = "172.16.31.4:2379"
+
```
Comment (TomShawn) [inline tidb-lightning/deploy-tidb-lightning.md:131]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**